src/terrain/routes/data_usage_api.clj

```diff
@@ -10,17 +10,20 @@
 (defn data-usage-api-routes
   []
   (optional-routes
-    [config/data-usage-api-routes-enabled]
+   [config/data-usage-api-routes-enabled]
 
-    (context "/resource-usage" []
-      :tags ["resource-usage"]
+   (context "/resource-usage" []
+     :tags ["resource-usage"]
 
-      (context "/data" []
-        (GET "/current" []
-          :middleware [require-authentication]
-          :summary schema/UserCurrentDataSummary
-          :description schema/UserCurrentDataDescription
-          :return schema/UserCurrentDataTotal
-          (ok (dua/user-current-usage (:username current-user))))))))
+     (context "/data" []
+       (GET "/current" []
+         :middleware [require-authentication]
+         :summary schema/UserCurrentDataSummary
+         :description schema/UserCurrentDataDescription
+         :return schema/UserCurrentDataTotal
+         (ok (dua/user-current-usage (:username current-user))))
+
+       (GET "/overage" []
+         :middleware [require-authentication])))))
```

Comment:
Not sure what's going on here? I'm honestly not sure what compojure will do with an empty route like this.

Reply:
I decided to remove it. It was originally going to be in there for debugging purposes, but it doesn't seem necessary.

Reply:
Also, thanks for catching that.